lib/brains/urls.py

```diff
@@ -2,15 +2,17 @@
 
 from rest_framework.routers import DefaultRouter
 
-from lib.brains.views import buyer
+from lib.brains.views import buyer, paymethod
 
 router = DefaultRouter()
 router.register(r'buyer', buyer.BraintreeBuyerViewSet, base_name='buyer')
```

Comment:
Yeah, then I'd feel like I had to move the tests and serializers around as well and someone changes the URL structure and all the folders are wrong and... sound ok, but feels a bit fiddy.